tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/io/CommandLine.java

```diff
@@ -103,6 +91,22 @@ public String execute(String command) throws IOException, InterruptedException {
     return this.execute(command, DEFAULT_TIMEOUT);
   }
 
+  private List<String> getShell() {
+    boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
+    String shellPath = System.getProperty("shellPath", System.getenv().getOrDefault("SHELL_PATH", ""));
+    List<String> shell;
+    if (!shellPath.isEmpty()) {
+      shell = Arrays.asList(shellPath, shellPath.contains("cmd.exe") ? "/c" : "-c");
```

Comment:
I'd probably just return the value instead of doing if/else. But that's super minor.